.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+services:
```

Comment:
This seems unrelated to the rest of the PR. I feel like this should be in a separate PR

Reply:
ohhh! I didn't want to commit that file.
Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/490/commits/e6593b88a83a5aa4e03a24b1ece01c77e89d429a.